src/pages/Account/CodeOfConduct.tsx

```diff
@@ -83,22 +80,21 @@ export const CodeOfConduct: React.FC = () => {
   );
 
   useEffect(() => {
-    if (!currentVenue) return;
+    if (!space) return;
 
     // @debt replace this with useCss?
-    updateTheme(currentVenue);
-  }, [currentVenue]);
+    updateTheme(space);
```

Comment:
Are these things really used? Maybe that above 
```tsx
// @debt replace this with useCss?
```
should be replaced by
```tsx
// @debt remove this
```

Reply:
No idea, most of the stuff I saw is unknown to me. We've been focus on admin for so long, at this point I don't know whats legacy and what's not. I'm afraid to touch anything

Reply:
Marking it in the refactor issue https://github.com/sparkletown/internal-sparkle-issues/issues/1462